usaspending_api/etl/es_award_template.json

```diff
@@ -416,30 +416,7 @@
           "type": "text"
         },
         "treasury_accounts": {
-          "type": "nested",
```

Comment:
One of the major benefits we were trying to get out of removing nested fields was allowing the sort to be put back in place, and the sort to default to the same sort that @emily-brents was putting into her default Advanced Search queries. 

@emily-brents took the sort out of the mapping [in her PR because the nested fields prohibited it](https://github.com/fedspendingtransparency/usaspending-api/pull/2259/files#r374893886). Here its left as-was which is:
```
  "sort": {
    "fields": [
      "total_obligation",
      "action_date"
    ],
    "order": [
      "desc",
      "desc"
    ]
  },
```
That's not the ideal sort because her PR uses `total_obligation` and `award_id`. 

We need to coordinate who will be getting the sort in, and using the right fields.

Reply:
Looks like in her 7.1 PR it's being set, but not to the fields that she needs I don't think:
https://github.com/fedspendingtransparency/usaspending-api/pull/2259/files#diff-570f89ea9b600b880909d51919150cdbR9

(I see `action_date` and not `award_id`)